python/pyarrow/tests/test_parquet.py

```diff
@@ -2306,6 +2307,34 @@ def test_backwards_compatible_column_metadata_handling(datadir):
     tm.assert_frame_equal(result, expected[['a']].reset_index(drop=True))
 
 
+@pytest.mark.pandas
+@pytest.mark.skipif(pickle.HIGHEST_PROTOCOL < 3,
```

Comment:
I'm not sure about which protocol has introduced support for instancemethods.

Reply:
Hmm... why it is trying to pickle an instance method?